client/components/pages/editor/sections/Hero.jsx

```diff
@@ -19,13 +19,8 @@ export default class Hero extends Component{
 
   handleChange = (field, value, index) => {
     const d = this.props.data
-
-    if (index != null){
-      d[field][index] = value
-
-    } else {
-      d[field] = value
-    }
+    if (index === undefined || index === null) d[field] = value
```

Comment:
tbh not a fan of leaving out the curly braces -- lends itself to bugs when new code is added